src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,52 @@
-import React from 'react';
+import React, { FormEvent, useState } from 'react';
 import './FindMovie.scss';
+import cn from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+interface Props {
+  onAddMovie: (movie: Movie) => void
+}
+
+export const FindMovie: React.FC<Props> = React.memo(({ onAddMovie }) => {
+  const [query, setQuery] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [error, setError] = useState(false);
+
+  const handleGetMovie = (event: FormEvent) => {
+    event.preventDefault();
+    setIsLoading(true);
+
```

Comment:
don't send a request if you have an invalid `title`, just with spaces, for example
Also - put an error like `Put the movie name`